src/components/UserPost/UserPost.jsx

```diff
@@ -0,0 +1,12 @@
+import './UserPost.css'
+
+function UserPost()
+{
+    return (
+        <div className='user-post'>
+
+        </div>
```

Comment:
Suggestion: if some file is empty and in future you are going to work on that file add some comment about what is there to be done before pushing , by this way reviewer will know that it's not unnecessary file

Reply:
Done, thanks for pointing it out